jsk_2016_01_baxter_apc/node_scripts/set_contents_param.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import sys
+
+import rospy
+
+
+if __name__ == '__main__':
+    rospy.init_node('set_contents_param')
+
+    json_file = rospy.get_param('~json')
+    contents = rospy.get_param('~contents')
+
+    data = json.load(open(json_file))
+    rospy.set_param('~param_name', data[contents])
```

Comment:
`~param` is enough following [rosparam tag](http://wiki.ros.org/roslaunch/XML/rosparam)?